packages/tldraw/src/lib/shapes/draw/getPath.ts

```diff
@@ -119,11 +119,17 @@ export function getPointsFromSegments(segments: TLDrawShapeSegment[]) {
 	return points
 }
 
-export function getDrawShapeStrokeDashArray(shape: TLDrawShape, strokeWidth: number) {
+export function getDrawShapeStrokeDashArray(
+	shape: TLDrawShape,
+	strokeWidth: number,
+	zoomLevel: number
+) {
 	return {
 		draw: 'none',
 		solid: `none`,
-		dotted: `0.1 ${strokeWidth * 2}`,
+		// If we're zoomed way out (10%), then we need to make the dotted line go to 9 instead 0.1
+		// Chrome doesn't render anything otherwise.
+		dotted: `${zoomLevel < 0.2 ? 0 : 0.1} ${strokeWidth * 2}`,
```

Comment:
Seems like it also disappears at 21 and sometimes even 22%? At least from my tests 23% works.

https://github.com/user-attachments/assets/6ad40f86-a9fc-404a-aebc-69c41ead9666






```suggestion
		dotted: `${zoomLevel < 0.23 ? 0 : 0.1} ${strokeWidth * 2}`,
```

Reply:
Browser scale is going to effect this too :(

Reply:
> Seems like it also disappears at 21 and sometimes even 22%? At least from my tests 23% works.

argh, that's worrisome then, that isn't consistent with when it's appearing/disappearing for me. fun fun fun.
@MitjaBezensek is that when it's on the non-retina screen maybe?

Reply:
Ah yes. On the laptop screen it seems at around 10% and on external, non-retina one its at 22% 🤷‍♂️

Reply:
Settling on 20%. we can dial this in

Reply:
nice! but yeah, if it's still happening from 20 < x < 23, how can else we improve this then?